fdbserver/RestoreWorker.actor.cpp

```diff
@@ -185,8 +185,10 @@ ACTOR Future<Void> monitorWorkerLiveness(Reference<RestoreWorkerData> self) {
 void initRestoreWorkerConfig() {
 	opConfig.num_loaders = g_network->isSimulated() ? 3 : opConfig.num_loaders;
 	opConfig.num_appliers = g_network->isSimulated() ? 3 : opConfig.num_appliers;
+	// TODO: Set the threshold to a random value in a range
 	opConfig.transactionBatchSizeThreshold =
 	    g_network->isSimulated() ? 512 : opConfig.transactionBatchSizeThreshold; // Byte
+	opConfig.batchSizeThreshold = g_network->isSimulated() ? 10 * 1024 * 1024 : opConfig.batchSizeThreshold; // Byte
```

Comment:
It can be. I'm still debating on how the `opConfig ` should be configured from DBA perspective.
The `opConfig` will be configured differently based on the size of cluster.
One choice is using knobs;
The other is letting DBA to configure this in a configuration file or in a special system key.

Maybe knob is an easier and more compatible to the current operation tools?